src/main/java/rcms/utilities/daqexpert/persistence/LogicModuleRegistry.java

```diff
@@ -89,6 +90,8 @@
 
 	FEROLFifoStuck		   		(new FEROLFifoStuck(),		       ConditionGroup.OTHER,                 "",                                                  500,  10500),
 
+	RuFailed                (new RuFailed(),                 ConditionGroup.OTHER,                 "",                                                  900,   9500),
```

Comment:
Yes, that's the difference between them, RuFailed is independent from other KnownFailure LMs, that's why I was confused why you want to run it later. Ok, waiting for the changes to merge the pull request.